apps/web/src/pages/quick-start/consts.tsx

```diff
@@ -234,3 +238,59 @@ export enum FlowTypeEnum {
   IN_APP = 'in_app',
   OTHER = 'other',
 }
+
+export const quickStartChannels: IQuickStartChannelConfiguration[] = [
+  {
+    Icon: Bell,
+    title: 'In-App notifications',
+    displayName: 'In-App',
+    type: ChannelTypeEnum.IN_APP,
+    description: 'A set of APIs and components to create a customized notification center',
+    clickHandler: (navigation: NavigateFunction) => {
+      if (navigation) {
+        navigation(ROUTES.GET_STARTED_NOTIFICATION);
+      }
+    },
+  },
+  {
+    Icon: Mail,
+    title: 'Email',
+    displayName: 'Email',
+    type: ChannelTypeEnum.EMAIL,
+    description: '🎉  Try our gift: 300 emails Use Novu provider for free or change the provider to yours',
+    clickHandler: () => {},
```

Comment:
this way it might be hard to integrate with a integrations store modal, like from here you are not able to say open modal

Reply:
From here we could say open modal if this handler will get 'setter' from the upper scope (like I did on another handler with 'navigation'), do you think it will work?